deploy/charts/rawfile-csi/templates/01-controller-plugin.yaml

```diff
@@ -42,6 +42,10 @@ spec:
           image: "{{ .Values.controller.image.repository }}:{{ .Values.controller.image.tag }}"
           imagePullPolicy: {{ .Values.controller.image.pullPolicy }}
           args:
+          {{- if regexMatch "\\/canonical\\/" .Values.controller.image.repository }}
+            - --args
+            - rawfile
+          {{- end }}
```

Comment:
`rockcraft` creates images with `pebble` as the entrypoint, setting args on the service requires these two extra arguments first

Reply:
instead of attempting to resolve this by looking at a prefix of the image, should we just be able to set the full `command` here instead via helm values?

Reply:
setting the args via helm!.  Done